### PR TITLE
feat(openrtb): add Loss reason

### PR DESCRIPTION
**Pull request by xiaopeng0216**
# Context
The PR add the LossReason in the openrtb

### CODE REVIEW

File: openrtb/lossreason.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	LossReasonInvalidDealID
+	LossReasonInvalidAuctionID
+	LossReasonInvalidAdvertiserDomain
+	LossReasonMissingMarkup
+	LossReasonMissingCreativeID
+	LossReasonMissingBidPrice
+	LossReasonMissingMinimumCreativeApprovalData
+
+	LossReasonBidWasBelowAuctionFloor = 100
+	LossReasonBidWasBelowDealFloor
```
Comment (Fan-Xu) [inline openrtb/lossreason.go]: Please use different value for different loss reason

File: openrtb/lossreason.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	LossReasonLostToHigherBid
+	LossReasonLostToABidForAPMPDeal
+	LossReasonBuyerSeatBlocked
+
+	LossReasonCreativeFilterGeneral = 200
+	LossReasonCreativeFilterPendingByExchange
```
Comment (Fan-Xu) [inline openrtb/lossreason.go]: same as above

**Comment by Fan-Xu** [comment]
LGTM

**Comment by xiaopeng0216** [comment]
LGTM  :trollface:

**Pull request merged**